app/Actions/Server/ValidateServer.php

```
@@ -45,6 +45,15 @@ public function handle(Server $server)
             throw new \Exception($this->error);
         }
 
+        $prerequisitesInstalled = $server->validatePrerequisites();
```

PHPMD (2.15.0)

Avoid excessively long variable names like $prerequisitesInstalled. Keep variable name length under 20. (undefined)

(LongVariable)